contracts/token/ERC721/extensions/draft-ERC721Votes.sol

```diff
@@ -27,112 +25,12 @@ import "../../../utils/cryptography/draft-EIP712.sol";
-    mapping(address => Counters.Counter) private _nonces;
-
-    /**
-     * @dev Initializes the {EIP712} domain separator using the `name` parameter, and setting `version` to `"1"`.
-     *
-     * It's a good idea to use the same `name` that is defined as the ERC721 token name.
-     */
-    constructor(string memory name, string memory symbol) ERC721(name, symbol) EIP712(name, "1") {}
-
-    /**
-     * @dev Emitted when an account changes their delegate.
-     */
-    event DelegateChanged(address indexed delegator, address indexed fromDelegate, address indexed toDelegate);
-
-    /**
-     * @dev Emitted when a token transfer or delegate change results in changes to an account's voting power.
-     */
-    event DelegateVotesChanged(address indexed delegate, uint256 previousBalance, uint256 newBalance);
-
-    /**
-     * @dev Get number of checkpoints for `account`.
-     */
-    function numCheckpoints(address account) public view virtual returns (uint32) {
-        return SafeCast.toUint32(_votes.getTotalAccountVotes(account));
-    }
-
-    /**
-     * @dev Get the `pos`-th checkpoint for `account`.
-     */
-    function checkpointAt(address account, uint32 pos) public view virtual returns (Checkpoints.Checkpoint memory) {
-        return _votes.getTotalAccountVotesAt(account, pos);
-    }
-
-    /**
-     * @dev Get the address `account` is currently delegating to.
-     */
-    function delegates(address account) public view virtual returns (address) {
-        return _votes.delegates(account);
-    }
-
-    /**
-     * @dev Gets the current votes balance for `account`
-     */
-    function getVotes(address account) public view returns (uint256) {
-        return _votes.getVotes(account);
-    }
-
-    /**
-     * @dev Retrieve the number of votes for `account` at the end of `blockNumber`.
-     *
-     * Requirements:
-     *
-     * - `blockNumber` must have been already mined
-     */
-    function getPastVotes(address account, uint256 blockNumber) public view returns (uint256) {
-        return _votes.getVotesAt(account, blockNumber);
-    }
-
-    /**
-     * @dev Retrieve the `totalVotingPower` at the end of `blockNumber`. Note, this value is the sum of all balances.
-     * It is but NOT the sum of all the delegated votes!
-     *
-     * Requirements:
-     *
-     * - `blockNumber` must have been already mined
-     */
-    function getPastVotingPower(uint256 blockNumber) public view returns (uint256) {
-        require(blockNumber < block.number, "ERC721Votes: block not yet mined");
-        return _votes.getTotalVotesAt(blockNumber);
-    }
-
-    /**
-     * @dev Delegate votes from the sender to `delegatee`.
-     */
-    function delegate(address delegatee) public virtual {
-        _delegate(_msgSender(), delegatee);
-    }
-
+abstract contract ERC721Votes is ERC721, Votes {
     /**
-     * @dev Delegates votes from signer to `delegatee`
+     * @dev Returns total amount of votes at given position.
```

Comment:
"Position" should be "block number", and note that the function argument is "timestamp" and should also be "blockNumber".